documentation/src/main/asciidoc/reference/mapper-orm-mapping-directfieldmapping.asciidoc

```diff
@@ -159,6 +159,15 @@ Whether the field can be <<search-dsl-projection, projected on>>,
 i.e. whether the field value is stored in the index to allow retrieval later when querying.
 +
 Value: `Projectable.YES`, `Projectable.NO`, `Projectable.DEFAULT`.
++
+The defaults are different for the <<backend-lucene,Lucene>> and <<backend-elasticsearch,Elasticsearch>> backends:
+with Lucene, the default is `Projectable.NO`, while with Elasticsearch it's `Projectable.YES`.
+
+[IMPORTANT]
+====
+For <<backend-elasticsearch,Elasticsearch>> if any of `projectable` or `sortable` properties are resolved to `YES`
+on a `GeoPoint` field then this field automatically becomes both `projectable` and `sortable` even if one of them was explicitly set to `NO`.
+====
 
```

Comment:
Good idea to mention this, but I wouldn't consider this "important": after all, if a user expects a field not to be sortable but it ends up being sortable, they won't see the difference.

```suggestion
+
[NOTE]
====
For <<backend-elasticsearch,Elasticsearch>> if any of `projectable` or `sortable` properties are resolved to `YES`
on a `GeoPoint` field then this field automatically becomes both `projectable` and `sortable` even if one of them was explicitly set to `NO`.
====
```